crates/sui-core/src/authority_active/gossip/mod.rs

```diff
@@ -63,14 +67,79 @@ const REFRESH_FOLLOWER_PERIOD_SECS: u64 = 60;
 
 use super::ActiveAuthority;
 
+#[derive(Clone, Debug)]
+pub struct GossipMetrics {
+    pub concurrent_followed_validators: IntGauge,
```

Comment:
Maybe a comment over the structure that says: for description of each metric see the `new` function.